Python/ceval.c

```diff
@@ -15,6 +15,7 @@
 #include "pycore_long.h"          // _PyLong_GetZero()
 #include "pycore_object.h"        // _PyObject_GC_TRACK()
 #include "pycore_moduleobject.h"  // PyModuleObject
+#include "pycore_opcode.h"        // EXTRA_CASES
```

Comment:
I don't know why you call out EXTRA_CASES in the comment -- this file uses several other tables defined there as well.

Reply:
When I add an ``#include``, I put the same of a single symbol get from this ``#include``, even if many other symbols are used. I only do that to check time to time if the include is still used: here I just have check if ``EXTRA_CASES`` is still used in the file. Otherwise, for each ``#include``, I have to remove the line, try to build, and check if the C file still builds successfully.

Reply:
Would be nice if there was tooling that did this instead. Without the comment.